src/tls.rs

```diff
@@ -32,7 +32,7 @@ pub(crate) enum TlsConfigError {
     /// An error from an empty key
     EmptyKey,
     /// An error from an invalid key
-    InvalidKey(TLSError),
+    InvalidKey(TlsError),
```

Comment:
I suppose this is technically a public API change... should this change bump the major version?

Reply:
The type is `pub(crate)`, so I don't think it's an issue.